public/app/plugins/datasource/testdata/nodeGraphUtils.ts

```diff
@@ -123,9 +123,9 @@ export function generateRandomNodes(count = 10) {
   for (const node of nodes) {
     nodeFields.id.values.add(node.id);
     nodeFields.title.values.add(node.title);
-    nodeFields.subTitle.values.add(node.subTitle);
-    nodeFields.mainStat.values.add(node.stat1);
-    nodeFields.secondaryStat.values.add(node.stat2);
+    nodeFields.subTitle?.values.add(node.subTitle);
```

Comment:
I assume this fixes the error but makes this code not actually do anything. If nodeFields does not contain subTitle field we should check why as it was meant to be populated with some data.

Reply:
I just some digging around and found the reason for this issue. The keys should be all lowercase, i.e. they should be `subtitle`, `mainstat` and `secondarystat`. See [nodeGraph.ts](https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/utils/nodeGraph.ts)

I have now changed the implementation so that the `NodeGraphDataFrameFieldNames` enum is used to access valid keys of `nodeFields`. See commit https://github.com/grafana/grafana/pull/54298/commits/f80340601fb61f56ddea9df83b9dab3d3be6cbb6. 

Alternatively, I could've simply changed the keys to lowercase, e.g. `nodeFields.subtitle`, but using the `NodeGraphDataFrameFieldNames` enum means that if the values of the keys of this enum are ever changed, things will still work as intended as we're getting the values from the enum keys rather than just copying the values (which could change).

This is much better than my initial "fix", which as you rightly assumed got rid of the errors in the console but didn't fix the underlying issue.